ConvivaIntegration/components/conviva/ConvivaAnalyticsTask.brs

```diff
@@ -74,6 +74,14 @@ sub monitorVideo()
         onStateChanged(data)
       else if field = "invoke"
         invoke(data)
+      else if field = "adBreakStarted"
```

Comment:
do we have some Enum values for those? I don't think so as they only exist in the yospace integration right?

Reply:
We do actually. I made them back when I updated the sample app. I will use it here.

Reply:
But there is none for "adError" since that field does not exists in the integration yet.